src/components/containers/CardTable.vue

```diff
@@ -0,0 +1,198 @@
+<template>
+    <Card
+        class="card-f card-full-width"
+        :loading="loading"
+        :error="error"
+    >
+        <template #title>
+            {{getNameByKey(title)}}
+        </template>
+        <template #control>
+            <div class="ex-infotext" v-if="hasInfoText"> {{infoText}} </div>
+            <Dropdown
+                v-if="hasFilter"
+                :options="filterOptions"
+                :value="filterValue"
+                right
+                @change="changeFilterValue"
+            />
+            <Pagination
+                v-else-if="pagination === 'custom'"
+                v-bind="paginationOptions"
+            />
+        </template>
+        <template #body>
+            <TableListView
+                v-if="Array.isArray(data)"
+                :data="data"
+                :timeline="manager"
+                :timelinePagination="pagination === 'server'"
+                :pagination="pagination === 'client'"
+                :pageSize="pageSize"
+            />
+            <TableInfoView
+                v-else-if="typeof data === 'object'"
+                :data="data"
+            />
+            <div v-else>
+                {{getNameByKey('No data provided')}}
+            </div>
+        </template>
+
+        <template #error>
+            {{getNameByKey(errorMessage)}}
+        </template>
+    </Card>
+</template>
+
+<script>
+import Card from '@/components/containers/Card.vue'
+import TableListView from '@/components/tables/TableListView.vue'
+import TableInfoView from '@/components/tables/TableInfoView.vue'
+import Dropdown from '@/components/controls/Dropdown.vue'
+import Pagination from '@/components/controls/Pagination.vue'
+
+export default {
+  components: {
+    Card,
+    TableListView,
+    TableInfoView,
+    Dropdown,
+    Pagination
+  },
+
+  props: {
+    // Displays title in the card header
+    title: {
+      type: String
+    },
+    // Displays text in the card header
+    hasInfoText: {
+      type: Boolean,
+      default: false
+    },
+    infoTextGetter: {
+      type: String
+    },
+    // Adds dropdown for Filter Data Manager
+    hasFilter: {
+      type: Boolean,
+      default: false
+    },
+    // Data Manager getter (DataSet, Timeline, Filter)
+    managerGetter: {
+      type: String
+    },
+    // Object or Array. If not provided, will use data from Data Manager
+    dataGetter: {
+      type: String
+    },
+    // pagination: 'disabled' | 'client' | 'server'(works only with Timeline data manager) | 'custom'(displayed in the header)
+    pagination: {
+      type: String,
+      default: 'disabled'
+    },
+    // options for custom pagination (props of the Parination component) (displayed in the header)
+    paginationOptions: {
+      type: Object,
+      default: () => ({})
+    },
+    // client-side pagination pageSize
+    pageSize: {
+      type: Number,
+      default: 10
+    },
+    // columns to show in the table
+    columns: {
```

Comment:
Just suggestion, do you think is good idea rename to `fields` it sound more neutral.
`columns` shoud be fine for data list (Array) but for `Object {}` sound wrong.

Reason: I added fliter for Object data as well, i call it `fields`.

Reply:
Yes, it makes sense